### PR TITLE
JS Capstone - API-based Webapp

### PR DESCRIPTION
**Pull request by Gegardus**
The JavaScript capstone project is about building a web application based on the external TVmaze API (data about TV series and movies) with two user interfaces. For work coordination was used Kanban board with filled-in tasks. In development process was implemented: 
- JavaScript to make websites dynamic and build basic single-page apps;
- ES6 syntax: modules, callbacks, promises, etc.;
- Webpack;
- Jest for unit tests;
- Gitflow;
- Sending and receiving data from an API.

### CODE REVIEW

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **TEAM,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Descriptive PR ✔️ 
- You have followed  JavaScript best practices. ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  });
+
+  const like = document.querySelectorAll('.like');
+  like.forEach((item) => {
+    item.addEventListener('click', () => {
+      const movieId = item.getAttribute('data-id');
+      if (item.style.color !== 'skyblue') {
+        item.style.color = 'skyblue';
+        item.firstElementChild.innerHTML = Number(item.firstElementChild.innerHTML) + 1;
+        addLike(movieId);
```
[the remark this answers is not in the document]
Comment (ahmetbozaci) [reply src/index.js:82]: Hello @moise10r, you mentioned wrong in the review.

[the remark this answers is not in the document]
Comment (Kandy-Peter) [reply src/index.js]: Hi @moise10r,
We don't think that it's really required. We also know that users can't like an item two or more times but we resolved that as you required.

[the remark this answers is not in the document]
Comment (Gegardus) [reply src/index.js]: @ahmetbozaci And what we should do now?

[the remark this answers is not in the document]
Comment (Gegardus) [reply src/index.js]: @ahmetbozaci what do you mean ?

[the remark this answers is not in the document]
Comment (ahmetbozaci) [reply src/index.js]: @moise10r  You should change the name that you mention so people can get a correct notification.
image

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada:  :champagne:  :confetti_ball: 
Hi  :wave:   :wave:  @Gegardus , @ahmetbozaci  and @Kandy-Peter 

200w 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your README file looks very professional :+1: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Gegardus** [comment]
Thanks.

File: src/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (moise10r) [reply src/index.js]: Yeah sure, I'll take more care, Thank you @ahmetbozaci !!!

[the remark this answers is not in the document]
Comment (moise10r) [reply src/index.js]: Yeah you can submit for a  second review :+1:

**Pull request merged**